inspirehep/dojson/hep/fields/bd1xx.py

```diff
@@ -175,7 +207,8 @@ def get_value(value):
         ]
         return {
             'a': value.get('full_name'),
-            'e': value.get('role'),
+            # FIXME: We only add the first role to the xml
+            'e': utils_get_value(value, 'contributor_roles[0].value'),
```

Comment:
I think `utils_get_value(value, 'contributor_roles.value')` would return you the whole list of values. (hopefully)